pallets/network-score/src/tests.rs

```diff
@@ -384,3 +384,130 @@ fn revoke_rating_with_existing_rating_identifier_should_fail() {
 		);
 	});
 }
+
+#[test]
+fn test_revise_rating_id_already_exists() {
```

Comment:
nit: Follow other naming conventions as in,
`fn revise_rating_with_existing_rating_identifier_should_fail()`